docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -34,13 +35,21 @@ class Book(UUIDAuditBase):
 sqlalchemy_config = SQLAlchemyAsyncConfig(
     connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config
```

Comment:
```suggestion
    connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config, create_all=True
```

Reply:
This flag will now work since we are swapping to the main `SQLAlchemyPlugin` here.